File: app/src/main/java/com/aquamorph/frcmanager/activities/SplashActivity.java
package com.aquamorph.frcmanager.activities;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

/**
 * <p></p>
 *
 * @author Christian Colglazier
 * @version 7/27/2016
 */
public class SplashActivity extends AppCompatActivity {

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);

		Intent intent = new Intent(this, MainActivity.class);
		startActivity(intent);
		finish();
	}
}

File: app/src/main/kotlin/com/aquamorph/frcmanager/activities/SplashActivity.kt
package com.aquamorph.frcmanager.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

/**
 * Splashscreen for app startup
 *
 * @author Christian Colglazier
 * @version 3/31/2018
 */
class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}